src/tests/admin/conferenceRoomResourceAssociations/AResourceCanBeAssociatedToARoomEnabled.java

```diff
@@ -0,0 +1,75 @@
+package tests.admin.conferenceRoomResourceAssociations;
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import framework.common.UIMethods;
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomInfoPage;
+import framework.pages.admin.conferencerooms.RoomResourceAssociationsPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+import framework.utils.readers.JsonReader;
+
+/**
+ * TC02: Verify that a resource can be associated to a room enabled.
```

Comment:
I think that it would be better if you change  a room enabled to an enabled room